silverback/middlewares.py

```diff
@@ -49,20 +44,23 @@ def fix_dict(data: dict, recurse_count: int = 0) -> dict:
         return message
 
     def _create_label(self, message: TaskiqMessage) -> str:
-        if labels_str := ",".join(f"{k}={v}" for k, v in message.labels.items()):
+        if labels_str := ",".join(
+            f"{k}={v}" for k, v in message.labels.items() if k != "task_name"
+        ):
             return f"{message.task_name}[{labels_str}]"
 
         else:
             return message.task_name
 
     def pre_execute(self, message: TaskiqMessage) -> TaskiqMessage:
-        if "task_type" not in message.labels:
-            return message  # Not a silverback task
+        # NOTE: Ensure we always have this, no matter what
+        message.labels["task_name"] = message.task_name
```

Comment:
issue: `message.task_name` won't exist on a message received by a worker.  That's why it was set as a label.

Reply:
Confused this with task_type.  ignore.

Reply:
I added that? bit confused on this comment